.github/workflows/dev-ci.yml

```diff
@@ -11,6 +11,15 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
+  get-pr:
+    # https://dev.to/suzukishunsuke/secure-github-actions-by-pullrequesttarget-641
+    outputs:
+        merge_commit_sha: ${{steps.pr.outputs.merge_commit_sha}}
+    runs-on: ubuntu-latest
```

Comment:
`ubuntu-latest` maybe opens a room for problems later, why not use a specific lts version? e.g. 22.04?

Reply:
This usually make no too much different because the action run on the specific environment (python or npm environment that created on top of the system). Using latest version without pinning to minor version makes it able to always use the new functionalities.

Just some interesting reading: https://snarky.ca/why-i-dont-like-semver/ 
I think it still under debate in all the community.

Reply:
It's alright, I guess. Just pointing out that the latest versions of Ubuntu might not be as stable as lts, which for whatever reason, might cause problems. 
But in any case, I agree, it's not much of a problem since we are just using python env.

Reply:
Thanks for sending the link btw, interesting :)